Debugging.md

```diff
@@ -0,0 +1,9 @@
+#Debugging

```

Comment:
The title is not showing correctly on github. It might be a space missing after #

Reply:
Added the space between the # and Debugging